roles/podman/tasks/main.yml

```diff
@@ -1,7 +1,27 @@
 ---
-- name: install podman
+- name: Ensure package is installed
   package:
     name: podman
     state: present
   tags:
     - role::podman
+
+- name: Ensure systemd service is present
+  template:
+    src: templates/container-.service.j2
+    dest: "/etc/systemd/system/container-{{ container.name }}.service"
+    owner: root
+    group: root
+    mode: "0644"
+  notify:
+    - reload systemd
+  tags:
+    - role::podman
+
+- name: Ensure the systemd service is started and enabled
+  systemd:
+    name: "container-{{ container.name }}.service"
+    state: started
+    enabled: true
+  tags:
+    - role::podman
```

Comment:
This should probably be part of a different task in here, otherwise it will try to do this everytime we run the role, which also gets slow for many services. If we have the `h-asgi` role depend on `podman`, and `podman`'s `main` tasks only install podman, then once we have multiple roles that depend on `podman`, they will only run the role the first time they need it. So my recommendation would be:

- `h-asgi` gets a dependency to `podman` in `meta/main.yml`
- `podman` gets a new task e.g. `setup-systemd-container.yml`
- the ``h-asgi`` tasks should include tasks from somewhere else

Reply:
I think i understand your general idea, but I am still a bit unsure what you are proposing. Could we take a call when you have time and talk over this?

Reply:
If we have these tasks in the base role like this, every dependency to the `podman` role with different `vars` will re-run the role again and thus also run all the tasks in here, such as installing podman itself. We should split the configuration into a separate task file, that is then included when we want to configure a container.

Reply:
We can also talk about this in a meeting, though. I'll message the channel shortly